timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -65,7 +66,7 @@ public static TimeLockResource create(MetricsManager metricsManager,
     }
 
     @Path("/lock")
-    public LockService getLockService(@Safe @PathParam("namespace") String namespace) {
+    public LockRpcClient getLockService(@Safe @PathParam("namespace") String namespace) {
```

Comment:
this is weird, this is on the server, and it shouldn't be an rpc client, it should be a service or the resource itself